src/graphics.c

```diff
@@ -3048,6 +3053,39 @@ struct word *word_new(const char *str)
 	w->unbrake = 0;
 	w->prerend = NULL;
 	w->hlprerend = NULL;
+
+	#ifdef _USE_HARFBUZZ
```

Comment:
It's like that it will be much cleaner to make static function named somehow detect_direction() and move logic there.